src/main/java/com/hubspot/maven/plugins/dependency/management/DependencyManagementAnalyzer.java

```diff
@@ -26,7 +33,23 @@ public boolean analyze() {
     boolean success = checkDependencyManagement();
     // don't combine with previous line, we don't want short-circuit evaluation
     success &= checkPluginManagement();
-
+    log.info("\n\n---- Issues found ----\n");
+    if (unmanagedDependencyError) {
+      log.warn(requireManagement.unmanagedDependencyMessage());
+    }
+    if (dependencyversionMismatchError) {
+      log.warn(requireManagement.dependencyVersionMismatchMessage());
+    }
+    if (unmanagedPluginError) {
+      log.warn(requireManagement.unmanagedPluginMessage()); }
```

Comment:
`}` on the next line

Reply:
sorry about these. Missed them in the previous commit